src/VscPreviewExtension/package.json

```diff
@@ -1,21 +1,29 @@
 {
   "name": "DfmPreview",
   "displayName": "DfmPreview",
-  "description": "Preview the DocFX Flavored Markdown in Visual Studio Code",
-  "version": "0.0.9",
+  "description": "Preview the DocFX Flavored Markdown and tokenTree in Visual Studio Code",
+  "version": "0.0.15",
   "publisher": "928pjy",
   "engines": {
     "vscode": "^1.3.0"
   },
   "categories": [
+    "Languages",
     "Other"
   ],
   "activationEvents": [
     "onCommand:DFM.showPreview",
     "onCommand:DFM.showPreviewToSide",
-    "oncommand:DFM.showSource"
+    "oncommand:DFM.showSource",
+    "onCommand:DFM.showTokenTreeToSide"
   ],
   "main": "./out/src/extension",
+  "icon": "img/dfm.png",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/928PJY/VscextensionText.git"
```

Comment:
how about remove repository first, after we prepare the pubishing cmd then add back

---
In reply to: [77289446](https://github.com/dotnet/docfx/pull/664#discussion_r77289446) [](ancestors = 77289446,77289337)